lib/exec.ml

```diff
@@ -118,12 +118,11 @@ let git_add_all_and_commit ~repo ~message () =
 let git_merge ?(args = Cmd.empty) ~from ~repo () = run_git ~repo Cmd.(v "merge" %% args % from)
 
 let git_resolve ~remote ~ref =
-  run_and_log_l Cmd.(v "git" % "ls-remote" % remote % ref) >>= function
-  | [ line ] -> Git.parse_ls_remote_line line >>= fun (commit, _) -> Ok { Git.Ref.t = ref; commit }
-  | [] -> Rresult.R.error_msgf "No %a ref for %s" Git.Ref.pp ref remote
-  | _ ->
-      Rresult.R.error_msgf "Ref %a is pointing to more than one commit for %s" Git.Ref.pp ref
```

Comment:
This case is now ignored: this might be a problem, right ?

Reply:
I doubt it can but I can bring it back to life if needed. Do you think this can actually happen?

Reply:
It might happen if a branch and a tag have the same name

Reply:
Good point, I'll fix that!

Reply:
I came with a quick fix. It's not perfect but it will do for now.

I think in the future we need to make it explicit whether we track a tag or a branch and eventually prompt the user when both are available and we have to choose.